homeassistant/components/tibber/sensor.py

```diff
@@ -671,6 +671,20 @@ async def _async_update_data(self) -> None:
                 self.hass.config_entries.async_reload(self.config_entry.entry_id)
             )
 
+    async def get_prices(self, tibber_home: tibber.TibberHome) -> dict:
```

Comment:
This function does not need to be part of the data coordinator?
Maybe better to keep it in services.py?

Reply:
Agree, will move this to service.py